DDDEastAnglia/Views/Shared/_Layout.cshtml

```diff
@@ -1,107 +1,108 @@
+                </a>
+                <div class="btn-group pull-right">
+                    @Html.Partial("_LoginPartial")
                 </div>
+                @Html.Action("RenderMenu", "NavigationBar")
             </div>
         </div>
+    </div>
+    <div class="container-fluid">
         <div class="container-fluid">
-            <div class="container-fluid">
-                <div class="row-fluid">
-                    <div class="span8">
-                        @RenderSection("featured", required: false)
-                        <div class="row-fluid">
-                            @RenderBody()
-                        </div>
-                    </div><!--/span9-->
-                    <div class="span4" id="sidebar">
-                        <div class="well">
-                            <section class="row-fluid">
-                                <h1>When and Where?</h1>
-                                <p>
-                                    @Html.Action("ConferenceDate", "Timeline")
-                                    <br />
-                                    @Html.Action("ConferenceTime", "Timeline")
-                                </p>
-                                <p>
-                                    <a href="http://westroad.org">West Road Concert Hall</a><br />
-                                    11 West Road<br />
-                                    Cambridge<br />
-                                    CB3 9DP
-                                </p>
-                            </section>
-
-                            @Html.Action("Details", "Timeline")
+            <div class="row-fluid">
+                <div class="span8">
+                    @RenderSection("featured", required: false)
+                    <div class="row-fluid">
+                        @RenderBody()
+                    </div>
+                </div><!--/span9-->
+                <div class="span4" id="sidebar">
+                    <div class="well">
+                        <section class="row-fluid">
+                            <h1>When and Where?</h1>
+                            <p>
+                                @Html.Action("ConferenceDate", "Timeline")
+                                <br />
+                                @Html.Action("ConferenceTime", "Timeline")
+                            </p>
+                            <p>
+                                <a href="http://westroad.org">West Road Concert Hall</a><br />
+                                11 West Road<br />
+                                Cambridge<br />
+                                CB3 9DP
+                            </p>
+                        </section>
 
-                            @Html.Partial("_Sponsors")
-                        </div>
+                        @Html.Action("Details", "Timeline")
+                        @{
+                            Html.RenderAction("Sidebar", "Sponsor");
```

Comment:
I think this is the only change in this file we actually care about that much. If you can, please try to keep functional changes separate from formatting changes.

Why does this call have to be in braces, but the line above does not?